go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/gorilla/handlers v1.4.2
 	github.com/gorilla/mux v1.7.3
 	github.com/gorilla/websocket v1.4.2 // indirect
+	github.com/jpillora/backoff v1.0.0
```

Comment:
Oh, it's a new backoff pkg. We are using `"k8s.io/client-go/util/flowcontrol"` in longhorn-manager repo. Not sure which one is better for this case

Reply:
I think `"k8s.io/client-go/util/flowcontrol"` is good enough, I would turn to use this pkg.